docs/MainSite/Installation.md

```diff
@@ -64,9 +63,10 @@ To build with the docker image:
    ```
    docker run -v SPECTRE_ROOT:SPECTRE_ROOT --name CONTAINER_NAME -i -t sxscollaboration/spectrebuildenv:latest /bin/bash
    ```
-   (The `--name CONTAINER_NAME` is optional, where CONTAINER_NAME is a name
-   of your choice. If you don't name your container, docker will generate an
-   arbitrary name.)
+   (The `--name CONTAINER_NAME` is optional, where CONTAINER_NAME is a
+   name of your choice. If you don't name your container, docker will
+   generate an arbitrary name.  `SPECTRE_ROOT:SPECTRE_ROOT` should be
+   two copies of the path `SPECTRE_ROOT`)
```

Comment:
This last sentence is a trivial statement, so I would prefer not to add it. Could you share your concern, so that we can explore other ways of improving the text to address it?

Reply:
At code review, we decided to define and export SPECTRE_ROOT and then this becomes $SPECTRE_ROOT:$SPECTRE_ROOT and we can remove the new sentence